compiler/include/AstLogger.h

```diff
@@ -30,7 +30,7 @@
 class AstLogger : public AstVisitor {
 public:
                  AstLogger();
-  virtual       ~AstLogger();
+  virtual       ~AstLogger() override;
```

Comment:
I think we should not have `virtual` and `override` on the same function?

Reply:
`virtual` is redundant but legal. I'll see about removing it.